From abc11fdc9288d5b5670a27bc1ab9354b71fbb69b Mon Sep 17 00:00:00 2001
From: Thomas Ackermann <th.acker@arcor.de>
Date: Tue, 9 Oct 2012 20:19:34 +0200
Subject: [PATCH] Change headline of technical/send-pack-pipeline.txt to not
 confuse its content with content from git-send-pack.txt

Signed-off-by: Thomas Ackermann <th.acker66@arcor.de>
---
 Documentation/technical/send-pack-pipeline.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/technical/send-pack-pipeline.txt b/Documentation/technical/send-pack-pipeline.txt
index 681efe4..9b5a0bc 100644
--- a/Documentation/technical/send-pack-pipeline.txt
+++ b/Documentation/technical/send-pack-pipeline.txt
@@ -1,5 +1,5 @@
-git-send-pack
-=============
+Git-send-pack internals
+=======================
 
 Overall operation
 -----------------
-- 
1.7.11.msysgit.1

